src/utils.cpp

```diff
@@ -1996,21 +2013,182 @@ void NodeCounter::operator -= (const NodeCounter& o)
     versions -= o.versions;
 }
 
-} // namespace
+SyncConfig::SyncConfig(std::string localPath,
+                       const handle remoteNode,
+                       const fsfp_t localFingerprint,
+                       std::vector<std::string> regExps,
+                       const Type syncType,
+                       const bool syncDeletions,
+                       const bool forceOverwrite)
+    : mLocalPath{std::move(localPath)}
```

Comment:
According to the Code style:
_Add a trailing comma to the last element in e.g. enums or maps for more practical diffs and merges._
Usually we put the comma at the end of the line, something like:
```suggestion
    : mLocalPath{std::move(localPath)},
```

Reply:
Nevermind. For initializer list, in case of multiple members, the only option is to add the comma at the beginning of the line, since the compiler will complain if there is an trailing comma.